docs/custom_container.md

```diff
@@ -7,12 +7,14 @@ to run test and deployment code.
 `mcr.microsoft.com/mlops/python:latest` is built with this
 [Dockerfile](../environment_setup/Dockerfile).
 
+**Note:** Additionally mcr.microsoft.com/mlops/python image is also tagged with amlsdk[version] e.g amlsdk1.1.5.1 and githubrelease[version] e.g. githubrelease131681 which helps in tracking the exact version of amlsdk and other python dependencies included in the image.
+
```

Comment:
Is 131681 release number?
Can we go with just mcr.microsoft.com/mlops/python:[githubreleaseversion] (e.g. mcr.microsoft.com/mlops/python:3.0.0) instead of mcr.microsoft.com/mlops/python:githubrelease[version]
(e.g. mcr.microsoft.com/mlops/python:githubreleaseversion3.0.0) ?

Reply:
amlsdk-[version]
githubreleaseversion-[version]